app/src/test/java/com/github/swent/echo/data/repository/RepositoryImplTest.kt

```diff
@@ -89,6 +89,32 @@ class RepositoryImplTest {
         assertEquals(Arrays.asList(event), eventsResult)
     }
 
+    @Test
+    fun joinEventTest() {
+        every {
```

Comment:
Tip: you could use `coEvery{}` and remove the `runBlocking`.

Reply:
thanks for the hint! I'll use that next time :)